### DISCUSSION TITLE
notification box doesn't show the whole message

### DISCUSSION THREAD

**Discussion by SiamSami** [linuxdeepin/developer-center#3024] [Features Request & Ideas | 特性请求 & 头脑风暴]
If a notification appears with a big description, then we can't read the whole message...
![deepinscreenshot_select-area_20190128154050](https://user-images.githubusercontent.com/33563064/51827287-39a55c80-2313-11e9-8208-1f76335f3f53.png)
![deepinscreenshot_select-area_20190128154133](https://user-images.githubusercontent.com/33563064/51827298-3f02a700-2313-11e9-8e52-5b0c8f689249.png)

**Comment by SiamSami** [comment]
@shule1987  It's been over a year and we still have this issue. When will it get fixed?

**Comment by BLumia** [comment]
Can confirm. `notify-send` a super long message to reproduce.

**Comment by meiyixiang** [comment]
his problem has always existed and has not been solved. I will feed it back to the product
